### PR TITLE
Fix stream parameter description on GZipStream, BrotliStream, and Def…

### PR DESCRIPTION
**Pull request by peto268**
## Summary

Fixes stream parameter description for ctors with CompressionLevel

Fixes #8271

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8276) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8276) before we can accept your contribution.<br/><br/>:x: peto268 [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=8276)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-api-docs?pullRequest=8276) it.</sub>

**Comment by gewarren** [comment]
@peto268 The link you referenced (#8271) is a PR, not an issue, so I'm not sure if any more changes are required here.

**Comment by peto268** [comment]
@gewarren The fixes made in #8271 were incomplete (some parameter descriptions were still wrong) so I referenced that.

File: xml/System.IO.Compression/BrotliStream.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
         <Parameter Name="compressionLevel" Type="System.IO.Compression.CompressionLevel" />
       </Parameters>
       <Docs>
-        <param name="stream">The stream to compress.</param>
+        <param name="stream">The stream to which compressed data is written.</param>
```
Comment (gewarren) [inline xml/System.IO.Compression/BrotliStream.xml:59]:
```suggestion
        <param name="stream">The stream to which compressed data is written or from which data to decompress is read.</param>
```

File: xml/System.IO.Compression/BrotliStream.xml [hunk 2]
```diff
@@ -56,7 +56,7 @@
...
         <param name="compressionLevel">One of the enumeration values that indicates whether to emphasize speed or compression efficiency when compressing data to the stream.</param>
```
Comment (gewarren) [inline xml/System.IO.Compression/BrotliStream.xml:60]:
```suggestion
        <param name="compressionLevel">One of the enumeration values that indicates whether to compress data to the stream or decompress data from the stream.</param>
```

**Review by gewarren — COMMENTED** [review]
Thanks for opening this PR @peto268. I think some additional changes are needed throughout - can you check?

File: xml/System.IO.Compression/BrotliStream.xml (hunk 1 above)
Comment (peto268) [reply xml/System.IO.Compression/BrotliStream.xml:59]: No, the ctors with CompressionLevel parameter create compression streams. Specifying CompressionLevel implies you want to compress.

File: xml/System.IO.Compression/BrotliStream.xml (hunk 2 above)
Comment (peto268) [reply xml/System.IO.Compression/BrotliStream.xml:60]: No, this is the compr. level, not mode.

**Review by gewarren — APPROVED** [review]

**Pull request merged**